src/Microsoft.IdentityModel.Tokens/JsonWebKeySet.cs

```diff
@@ -41,14 +42,16 @@ namespace Microsoft.IdentityModel.Tokens
     [JsonObject]
     public class JsonWebKeySet
     {
+        private readonly bool _skipUnresolvedJsonWebKeys = SkipUnresolvedJsonWebKeys;
```

Comment:
I didn't quite explain myself here. What I was suggesting is having the property:
public static bool DefaultSkipUnresolvedJsonWebKeys = true;

Then the instance is

public bool SkipUnresolvedJsonWebKeys {get;set;} = DefaultSkipUnresolvedJsonWebKeys;

Reply:
Got it. Addressed with 35fdd53974abd1f8330db12ab30b4898b60d94e7